content/questions/foundations/question-106.md

```diff
@@ -0,0 +1,15 @@
+---
+archetype: "questions"
+title: "Question 106"
+question: "<What are the different levels of GitHub Administration hierarchy?  (Choose 3)>"
+draft: false
+---
+
+> https://learn.microsoft.com/en-us/training/modules/github-introduction-administration/2-what-is-github-administration
+- [x] Team level
+- [x] Organization level
+- [ ] Personal level
+> (optional) GitHub Personal is a type of account and not an organizational hierarchy on GitHub.
+- [ ] Pro level
+> (optional) GitHub Pro is a type of account and not an organizational hierarchy on GitHub.
+- [x] Enterprize level
```

Comment:
```suggestion
- [x] Enterprise level
```

@FidelusAleksander  Oddly enough, the spelling test didn't catch this.

Reply:
Indeed, quite odd :man_shrugging: